lib/gutentag.rb

```diff
@@ -10,7 +10,7 @@ def self.dirtier=(dirtier)
   end
 
   def self.normaliser
-    @normaliser ||= Gutentag::TagName
+    @normaliser ||= -> (t){ t.to_s.downcase }
```

Comment:
should run the tests on 1.9 too if we want to keep compatibility ...used to writing this so I did not even notice ...